particles/Tutorial/Javascript/Demo/source/TicTacToeBoard.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
+
+/* global defineParticle */
+
+defineParticle(({SimpleParticle, html, log}) => {
+
+const template = html`
+  <style>
+    .grid-container {
+      display: grid;
+      grid-template-columns: 50px 50px 50px;
+      grid-column-gap: 0px;
+    }
+
+    .valid-butt {
+      border: 1px outset blue;
+      height: 50px;
+      width: 50px;
+      cursor: pointer;
+      background-color: lightblue;
+    }
+
+    .valid-butt:hover {
+      background-color: blue;
+      color: white;
+    }
+  </style>
+  <div class="grid-container">
+      <div class="grid-container">{{buttons}}</div>
+  </div>
+  <template button>
+    <button class=valid-butt type="button" on-click="onClick" value={{value}} \>
```

Comment:
Always quote attribute-values (`class="valid-butt"`) (ahem).

Also, `value` and `hidden` in this template.